src/main/java/org/owasp/esapi/logging/slf4j/Slf4JLogFactory.java

```diff
@@ -69,7 +71,8 @@ public class Slf4JLogFactory implements LogFactory {
         boolean logApplicationName = ESAPI.securityConfiguration().getBooleanProp(LOG_APPLICATION_NAME);
         String appName = ESAPI.securityConfiguration().getStringProp(APPLICATION_NAME);
         boolean logServerIp = ESAPI.securityConfiguration().getBooleanProp(LOG_SERVER_IP);
-        SLF4J_LOG_APPENDER = createLogAppender(logUserInfo, logClientInfo, logServerIp, logApplicationName, appName);
+        boolean omitEventTypeInLogs = ESAPI.securityConfiguration().getBooleanProp(OMIT_EVENT_TYPE_IN_LOGS);
```

Comment:
Same comment here as for `JavaLogFactory` class... for reasons noted in a comment in the LogPrefixAppender CTOR that you modifed, there is a (probably good) possibility that this property will not be found at all in the **ESAPI.properties** file, so you need to be prepared to catch `ConfigurationException` here and set `omitEventTypeInlogs` to `false` should that happen. Because we can't let a `ConfigurationException` propagate back to the caller if **Logger.OmitEventTypeInLogs** is not found. That would result in breaking people's code and as a result is just not acceptable.